{{ cookiecutter.repo_name }}/src/{{ cookiecutter.package_dir_name }}/__init__.py

```diff
@@ -1,4 +1,23 @@
-from ._version import get_versions
+#!/usr/bin/env python
+##############################################################################
+#
+# (c) 2024 The Trustees of Columbia University in the City of New York.
+# All rights reserved.
+#
+# File coded by: {{ cookiecutter.author_name }}
+#
+# See AUTHORS.rst for a list of people who contributed.
+# See LICENSE.rst for license information.
+#
+##############################################################################
 
-__version__ = get_versions()["version"]
-del get_versions
+"""Smaller shared functions for use by other diffpy packages.
```

Comment:
This needs to be changed to maybe `"""{{ cookiecutter.project_description }}`"""

Reply:
Yes. `{{ cookiecutter.project_short_description }}`